default.nix

```diff
@@ -6,6 +6,7 @@ in
 , mach-nix ? import sources.mach-nix { inherit pkgs pypiData; }
 , tahoe-lafs-source ? "tahoe-lafs"
 , tahoe-lafs-repo ? sources.${tahoe-lafs-source}
+, ...
```

Comment:
Does this still work with nix 2.3  (given that both nixos 21.05 and 21.11 use that version by default, we either need to support it for this file, or update what we deploy with in PrivateStorageio). I'm not opposed to bumping the version of nix used in our deployment, but if we do, we should probably verify that before making a change here that depends on it.

Reply:
Still just trying to figure out if it is possible to get a working configuration on CI without rolling our own Docker images or something nasty like that. :/

Reply:
For what it's worth, the thing that is broken on Nix 2.3 is that you cannot pass an unnamed argument through `...` from the command line.  So, for example, on Nix 2.3 you cannot override `tahoe-lafs-source` in `default.nix` when running `tests.nix`.

I also had a `python` argument in `default.nix` at one point (not in this branch, probably in the `python3` branch) which would be ignored.  I don't know if we're going to end up supporting more than one version of Python any time soon (since we'll drop Python 2 in the `python3` branch) so this might end up being not a particularly important use-case.